extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -65,10 +68,19 @@ class AmpLightbox extends AMP.BaseElement {
       right: 0,
     });
 
+    if (this.isScrollable_) {
+      st.setStyles(this.element, {
+        overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
not sure if this has been brought up but any child elements under parent element with -webkit-overflow-scrolling: touch that are display: relative or absolute do not display properly on ios - was having trouble with this using an amp-iframe with responsive layout in an amp-lightbox